bower.json

```diff
@@ -21,23 +21,23 @@
     "tests"
   ],
   "dependencies": {
-    "angular": "~1.2.29",
-    "angular-sanitize": "~1.2.29",
-    "angular-animate": "~1.2.29",
+    "angular": "~1.4",
+    "angular-sanitize": "~1.4",
+    "angular-animate": "~1.4",
     "angular-ui-router": "~0.2.10",
     "angular-uuids": "~0.0.3",
-    "angular-bootstrap": "0.12.0",
+    "angular-bootstrap": "1.3",
     "bootstrap-sass-official": "~3.3.1",
     "ngstorage": "~0.3.0",
     "moment": "~2.7.0",
     "lodash": "~2.4.1",
     "font-awesome": "~4.2"
   },
   "devDependencies": {
-    "angular-mocks": "~1.2.29"
+    "angular-mocks": "~1.4"
   },
   "testPath": "test/spec",
   "resolutions": {
-    "angular": "1.2.29"
+    "angular": ">=1 <1.4"
```

Comment:
I thought we upgraded to Angular 1.4, not 1.3; also, that `angular-bootstrap` v1.3 needed Angular >=1.4.